SETUP.md

```diff
@@ -77,8 +77,14 @@ It is best to copy and paste it.
 ### Correct username and password does not work
 Continuum depends on
 [Android System Webview](https://play.google.com/store/apps/details?id=com.google.android.webview)
-by default for logging into Reddit. So if having the login issue, your best
-course of action would be to upgraded to the latest version of Android
+by default for logging into Reddit. So if having the login issue, your quickest
+course of action would be to use the [login button](/assets/screenshots/login_button.png) located at the bottom of your screen which opens up reddit in your default browser for you to login. Keep in mind this method tends to not work with hardend browsers like Fennec. Here are some of the browsers which are guarenteed to work with this method; 
+
+- Brave
+
+-
```

Comment:
This is a stray bullet point. I also wouldn't just remove this. Bullet points only make sense if it is a list, and it is only a list with two or more items. So either reformat `Brave`, or add one or more browsers to your list. I would say `Chrome` is the best choice given it has no adblocking. Not that it would probably be an issue in this case, but I have repeatedly over years had weird compatibility issues with `Brave` both with `Android` and `Linux`.